examples/run_onepulse_nutation.py

```diff
@@ -0,0 +1,91 @@
+"""
+One Pulse Experiment
+====================
+
+Not an echo detection but rather just capturing
+the FID with a 4-step phase cycle. 
+"""
+
+from pyspecdata import getDATADIR
+from numpy import r_, linspace
+import sys
+import os
+import SpinCore_pp
+import SpinCore_pp as spc
+from SpinCore_pp.ppg import generic
+from datetime import datetime
+
+my_exp_type = "ODNP_NMR_comp/nutation"
+assert os.path.exists(getDATADIR(exp_type=my_exp_type))
+desired_p90_range_us = linspace(0.5, 15, 20, endpoint=False)
+# {{{importing acquisition parameters
+config_dict = SpinCore_pp.configuration("active.ini")
+(
+    nPoints,
+    config_dict["SW_kHz"],
+    config_dict["acq_time_ms"],
+) = spc.get_integer_sampling_intervals(
+    SW_kHz=config_dict["SW_kHz"],
+    time_per_segment_ms=config_dict["acq_time_ms"],
+)
+# }}}
+# {{{add file saving parameters to config dict
+config_dict["type"] = "FID_nutation"
+config_dict["date"] = datetime.now().strftime("%y%m%d")
+config_dict["FID_nutation_counter"] += 1
```

Comment:
Not for this pull request, but if you're ambitious next time this comes up: I've realized that the way we handle counters is ridiculous.  We should just modify the config parser so that it automatically keeps track of a counter for each "type".  Then we keep all the counters in their own section, and have a special function to increment the counter for each types. It should not be required to modify every time we want a counter with a new name.

Reply:
I agree on the idea here but also agree to save for another PR